scss/partials/_cards.scss

```diff
@@ -0,0 +1,101 @@
+/** TABLE OF CONTENT-Cards
+ — — — — — — — — — — — — — — — — — — — — — — — — — — — — — — /
+1 — Content Block
+
+/// END TABLE OF CONTENTS **/
+
+.content-block {
+  padding-right: 1.8rem;
+  position: relative;
+  width: calc(100vw - 2.8rem);
+
+  @include breakpoint(tablet) {
+    //TODO: remove margin on last card
+    padding-right: 7.5rem;
+    width: calc(100vw - 8.5rem);
+  }
+
+  &__button {
+    display: none;
+    @include breakpoint(tablet) {
+      display: inline-block;
+    }
+  }
+
+  &__date,
+  &__image {
+    align-self: center;
+    grid-row: 1 / 2;
+  }
+
+  &__date {
+    align-items: center;
+    background-color: $secondary-blue-1;
+    color: $primary-white;
```

Comment:
Would it be beneficial to use class `section--bg-sec-blue` for content-block__date instead or it doesn't really matter? @fienen  @nnedevn